tests/repros/1174/chat/package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "chat",
+  "version": "1.0.0",
+  "peerDependencies": {
+    "container": "^1.0.0",
+    "base": "^1.0.0"
+  },
```

I also have these dependencies included in `devDependencies` without which editing the type files has odd behavior in vscode:

```suggestion
  "peerDependencies": {
    "container": "^1.0.0",
    "base": "^1.0.0"
  },
  "devDependencies": {
    "container": "workspace:1.0.0",
    "base": "workspace:1.0.0",
  },
```